### PR TITLE
[Communication SDK for Android]: Bump up version number for communication-common and communication-chat

### PR DESCRIPTION
**Pull request by JianpingChen**

### CODE REVIEW

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Release History
+
+## 1.0.0-beta.3 (Unreleased)
```
Comment (vcolin7) [inline sdk/communication/azure-communication-common/CHANGELOG.md:3]: Should this say `1.0.0-beta.2`?

File: sdk/communication/azure-communication-common/build.gradle [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ext.publishName = "Microsoft Azure Android Client Library common code For Communication Service"
 description = "This package contains the Android client library common code for Microsoft Azure Communication Service."
-version = "1.0.0-beta.1"
+version = "1.0.0-beta.3"
```
Comment (vcolin7) [inline sdk/communication/azure-communication-common/build.gradle:3]: Should this say  `1.0.0-beta.2`?

Comment (JianpingChen) [reply sdk/communication/azure-communication-common/build.gradle:3]: beta.2 was the version we just published for communication in general. We want the current to be beta.3 (Unreleased). But Common is a special case where we didn't release beta.2. So the version has to jump to be consistent on next release.

File: sdk/communication/azure-communication-common/CHANGELOG.md (hunk 1 above)
Comment (JianpingChen) [reply sdk/communication/azure-communication-common/CHANGELOG.md:3]: Common is an odd ball. We didn't release beta.2 but we do want next release to be consistent with the others, i.e. next should be beta.3 (Unreleased)

Comment (RezaJooyandeh) [reply sdk/communication/azure-communication-common/CHANGELOG.md:3]: @vcolin7 JP is trying to sync this repo's version with the other languages.

**Review by RezaJooyandeh — APPROVED** [review]

**Review by bsiegel — APPROVED** [review]

**Comment by JianpingChen** [comment]
It seems there is a problem with CI. Victor's PR is suffering the same thing. I'll wait a bit to see if "/azp run" can fix it.

**Comment by JianpingChen** [comment]
@weshaggard Can you please play the same magic you did for Victor's PR? This is in the same trouble I believe.

**Pull request merged**